### PR TITLE
[SSSTARS] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: modules/rosapps/applications/screensavers/ssstars/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
- * PURPOSE:     Romanian Language File for Starfield ScreenSaver
- * TRANSLATOR:  Ștefan Fulea (stefan dot fulea at mail dot com)
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
  */
-
+ 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
```
Comment (binarymaster) [inline modules/rosapps/applications/screensavers/ssstars/lang/ro-RO.rc:9]:
```suggestion

LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
```

File: modules/rosapps/applications/screensavers/ssstars/lang/ro-RO.rc [hunk 2]
```diff
@@ -1,60 +1,60 @@
 /*
  * PROJECT:     ReactOS Starfield ScreenSaver
```
Comment (binarymaster) [inline modules/rosapps/applications/screensavers/ssstars/lang/ro-RO.rc:2]:
```suggestion
 * PROJECT:     Star field screensaver
```

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**